index.html

```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
+                </ul>
+                <ul>
+                    <li class="seeproject1">
+                        <h3 class="seeprojecttext">See Project</h3>
+                        <a></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="project1">
+                <h2 class="project1h">Professional Art Printing Data</h2>
+                <p class="project1p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project1tags">
+                    <li class="htmlbuttp">
+                        <p class="icontextp">html</p>
+                        <a></a>
+                    </li>
+                    <li class="bootstrapbuttp">
+                        <p class="icontextp">bootstrap</p>
+                        <a></a>
+                    </li>
+                    <li class="rubybuttp">
+                        <p class="icontextp">Ruby</p>
+                        <a></a>
+                    </li>
+                </ul>
+                <ul>
+                    <li class="seeproject1">
+                        <h3 class="seeprojecttext">See Project</h3>
+                        <a></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="project1">
+                <h2 class="project1h">Professional Art Printing Data</h2>
+                <p class="project1p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project1tags">
+                    <li class="htmlbuttp">
+                        <p class="icontextp">html</p>
+                        <a></a>
+                    </li>
+                    <li class="bootstrapbuttp">
+                        <p class="icontextp">bootstrap</p>
+                        <a></a>
+                    </li>
+                    <li class="rubybuttp">
+                        <p class="icontextp">Ruby</p>
+                        <a></a>
+                    </li>
+                </ul>
+                <ul>
+                    <li class="seeproject1">
+                        <h3 class="seeprojecttext">See Project</h3>
+                        <a></a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+    </section>
+    <section class="aboutme">
+        <div class="aboutme1">
+            <h2 class="aboutmeh">About me</h2>
+            <p class="aboutmep">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul>
+                <li class="resume">
+                    <h3 class="resumeh">Get My Resume</h3>
+                    <a></a>
+                </li>
+            </ul>
+        </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![aaa](https://user-images.githubusercontent.com/69638013/159135062-e452dc24-d825-4aa8-a211-2479791ef0c2.png)  | ![aaaa](https://user-images.githubusercontent.com/69638013/159135129-aac9acf0-b507-4523-bb2c-b9123f25608e.png)


- [ ] Kindly align the spacing in this section please check screenshot above ⬆️